ui/src/pages/backoffice/Patron/PatronDetails/components/ItemsSearch/ItemsSearch.js

```diff
@@ -1,45 +1,32 @@
-import React, { Component } from 'react';
+import { recordToPidType } from '@api/utils';
+import { Error, Loader, SearchBar } from '@components';
+import isEmpty from 'lodash/isEmpty';
 import PropTypes from 'prop-types';
-import {
-  Container,
-  Grid,
-  Segment,
-  Icon,
-  Header,
-  Button,
-  Input,
-} from 'semantic-ui-react';
-import { Error, Loader } from '@components';
+import React, { Component } from 'react';
+import { Container, Grid, Header, Icon, Segment } from 'semantic-ui-react';
 import { ItemsResultsList } from './components';
-import isEmpty from 'lodash/isEmpty';
-import { ES_DELAY } from '@config';
-import { recordToPidType } from '@api/utils';
 
 export default class ItemsSearch extends Component {
   constructor(props) {
     super(props);
     this.fetchItems = this.props.fetchItems;
-    this.updateQueryString = this.props.updateQueryString;
     this.clearResults = this.props.clearResults;
     this.checkoutItem = this.props.checkoutItem;
-    this.fetchUpdatedCurrentLoans = this.props.fetchUpdatedCurrentLoans;
+    this.updateQueryString = this.props.updateQueryString;
+    // this state is needed for the paste action,
+    // because components gets updated via input change
+    // but we need altered behaviour for the paste action
+    // and in this way the state from before update is preserved
```

Comment:
This is more of a general observation :), I am not sure we have talked about frontend documentation, but I see around JsDocs comments, maybe this can go in a doc comment for the class. We can address it at one of the stand ups, when and if we write docs in code.

Reply:
yes sure, I'm in favour of adding docs! thanks for bringing it up.
for now I was thinking it makes sense to add a comment when I write something unusual which is not obvious to understand. But IMO in this particular case it is not a class doc matter, it is very specific to that state.